README.md

```diff
@@ -279,7 +279,8 @@ Forks of this project demonstrate how to extend and integrate with other librari
  - https://github.com/vyakymenko/angular2-seed-express - integration with [Express](https://expressjs.com/) full-stack development.
  - https://github.com/UIUXEngineering/angular2-jspm-typescript-seed - integration with [JSPM](http://jspm.io/).
  - http://ngbot.io - a chat bot built with angular-seed.
-
+ - https://github.com/DmitriyPotapov/angular-seed-inspinia - integration with custom design template
```

Comment:
Change to `[angular-seed-inspinia](https://github.com/DmitriyPotapov/angular-seed-inspinia)`?